gulpfile.js

```diff
@@ -512,10 +512,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html", PATHS.dist + "/**/*.json"])
     .pipe(replace('[ios.latest-os-version]', '15.5'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.22.1'))
+    .pipe(replace('[ios.current-app-version]', '2.23.1'))
```

Comment:
```suggestion
    .pipe(replace('[ios.current-app-version]', '2.23.3'))
```

See https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.23.0-rc.10

Reply:
@Ein-Tim 2.23.**2** !

Reply:
@dsarkar You're absolutely right! Sorry, I fixed that.

Reply:
@Ein-Tim Thanks!